packages/suite/src/components/suite/Translation/components/HelperTooltip/Container.ts

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
```

Comment:
maybe we should use `useSelector` here instead of  redux container?

Reply:
Would be nicer indeed but I think we'll have to go over the container usages anyway at some point.

Reply:
I created a separate file for container because I really needed to separate the dummy UI component from its redux-connected counterpart as I am using both (dummy component is used in landing page, redux connected in suite). If I would keep it in one file the redux part would start complaining that redux store can't be find (because there is no redux in landing page)

Reply:
ohh, but it is true that I could use `useSelector` in this separate file. 👍